src/components/Container/Container.js

```diff
@@ -17,16 +17,17 @@ const Container = props => {
       />
     ));
 
+    const loadingComponents = [<MovieLoader key={0}/>, <MovieLoader key={1}/>, <MovieLoader key={2}/>]
+
     return (
       <main>
         <section className='movie-grid'>
-        {props.movies.length ? movieComponents :
-          <p>Loading...</p>}
-          </section>
+          {props.movies.length ? movieComponents : loadingComponents}
```

Comment:
I'm glad that the loading components lets us use conditional rendering in a more clean way.